index.md

```diff
@@ -1,22 +1,50 @@
 ---
 
 layout: col-sidebar
-title: OWASP wrongsecrets
-tags: example-tag
+title: OWASP WrongSecrets
+tags: wrongsecrets
 level: 2
-type: 
-pitch: A very brief, one-line description of your project
+type: tool
+pitch: Examples with how to not use secrets
 
 ---
+![Github Stars](https://img.shields.io/github/stars/commjoen/wrongsecrets?label=Stars%20WrongSecrets&style=social)
+[![OWASP Incubator Project](https://img.shields.io/badge/OWASP-Incubator%20project-48A646.svg)](https://owasp.org/projects/)
+[![Release version](https://img.shields.io/github/v/release/commjoen/wrongsecrets)](https://github.com/commjoen/wrongsecrets/releases/latest)
 
-This is an example of a Project or Chapter Page.  Please change these items to indicate the actual information you wish to present.  In addition to this information, the 'front-matter' above this text should be modified to reflect your actual information.  An explanation of each of the front-matter items is below:
+OWASP Wrongsecrets is the first Secrets Management focused vulnerable/p0wnable app! It can be used in security trainings, awareness demo's and as a testing environment for secret detection tools, and bad practice detection tooling.
```

Comment:
```suggestion
OWASP WrongSecrets is the first Secrets Management-focused vulnerable/p0wnable app! It can be used in security trainings, awareness demos, as a test environment for secret detection tools, and bad practice detection tooling.
```